test/features/template.scenario.feature

```diff
@@ -5,7 +5,7 @@ Scenario: Health Check
   When I get 'Health Check' response
   Then The status should be 'OK'
 
-Scenario: Hello
+Scenario: CreateThirdpartyTransactionRequests
```

Comment:
let extract this Scenario from `template` to external file dedicated to `transactionRequests`

Reply:
Ok extracted to transactionRequests file